src/Http/StringStream.php

```diff
@@ -266,4 +270,20 @@ public function getMetadata($key=null)
         }
         return $metadata[$key];
     }
+
+    /**
+     * A StringStream can act as a handler, which resolves to a response with
+     * itself as a body.
+     *
+     * This is primarily to allow it to be used content by middleware.
```

Comment:
Don't need the word `content` here -- or the sentence needs to be reworked.

Reply:
I think it was just missing the word "as".. is it better now?